pkg/osquery/table/onepassword_config.go

```diff
@@ -43,45 +71,60 @@ func (o *onePasswordAccountConfig) generateForPath(ctx context.Context, path str
 
 	db.Exec("PRAGMA journal_mode=WAL;")
 
-	rows, err := db.Query("SELECT user_email FROM accounts")
+	rows, err := db.Query("SELECT user_email, team_name, user_first_name, user_last_name, account_type FROM accounts")
 	if err != nil {
 		return nil, errors.Wrap(err, "query rows from onepassword account configuration db")
 	}
 	defer rows.Close()
 
 	var results []map[string]string
 	for rows.Next() {
-		var email string
-		if err := rows.Scan(&email); err != nil {
+		var email, team, firstName, lastName, accountType string
+		if err := rows.Scan(&email, &team, &firstName, &lastName, &accountType); err != nil {
 			return nil, errors.Wrap(err, "scanning onepassword account configuration db row")
 		}
-		if email == "" {
-			continue
-		}
-		results = addEmailToResults(email, results)
+		results = append(results, map[string]string{
+			"user_email":      email,
+			"username":        fileInfo.user,
```

Comment:
this username comes from the path on the computer, not from the 1password config correct?

Reply:
Correct. It's consistent with the rest of the tables.

Reply:
makes sense

Reply:
yes